sys/net/application_layer/dns/dns.c

```diff
@@ -125,7 +125,7 @@ static int _parse_dns_reply(uint8_t *buf, size_t len, void* addr_out, int family
             return tmp;
         }
         bufpos += tmp;
-        if ((bufpos + RR_TYPE_LENGTH + RR_CLASS_LENGTH + RR_TTL_LENGTH) >= buflim) {
+        if ((bufpos + RR_TYPE_LENGTH + RR_CLASS_LENGTH + RR_TTL_LENGTH + sizeof(uint16_t)) >= buflim) {
```

Comment:
Please fix the line-length in accordance to the coding conventions (100 chars max). E.g. like this
```suggestion
        if ((bufpos + RR_TYPE_LENGTH + RR_CLASS_LENGTH +
             RR_TTL_LENGTH + sizeof(uint16_t)) >= buflim) {
```

Reply:
Done, thank you! Should I squash?